projects/1.18/assets/caged-mobs/cagedmobs/lang/zh_cn.json

```
@@ -0,0 +1,66 @@
+{
+	"itemGroup.cagedmobstab": "笼中生物",
+	
+	"item.cagedmobs.dnasampler": "DNA采样器",
+	"item.cagedmobs.dnasamplerdiamond": "钻石DNA采样器",
+	"item.cagedmobs.dnasamplernetherite": "下界合金DNA采样器",
+
+	"item.cagedmobs.dnasampler.empty": "攻击生物以提取DNA样本",
+	"item.cagedmobs.dnasampler.tier1Info": "可对一般的生物进行采样",
+	"item.cagedmobs.dnasampler.tier2Info": "可对较强的生物进行采样",
+	"item.cagedmobs.dnasampler.tier3Info": "可对Boss进行采样",
+	"item.cagedmobs.dnasampler.makeEmpty": "Shift+右击以清除存储的DNA样本",
+	"item.cagedmobs.dnasampler.getBackEntity": "右击笼子以取回存储的DNA样本",
+	"item.cagedmobs.dnasampler.not_cachable": "无法对此类生物进行采样",
+	"item.cagedmobs.dnasampler.not_sufficient": "该采样器的级别不足以对该生物进行采样",
+	"item.cagedmobs.dnasampler.unknown_entity": "未知生物",
+
+	"item.cagedmobs.cookingupgrade": "笼升级：烹饪",
+	"item.cagedmobs.lightningupgrade": "笼升级：闪电",
+	"item.cagedmobs.arrowupgrade": "笼升级：箭矢",
+	"item.cagedmobs.cookingupgrade.info": "自动烹饪可烹饪的生物掉落物",
+	"item.cagedmobs.lightningupgrade.info": "用闪电杀死笼中的生物",
+	"item.cagedmobs.arrowupgrade.info": "用箭矢杀死笼中的生物",
+
+	"item.cagedmobs.upgrades.attach": "可以安装到生物笼上",
+
+	"item.cagedmobs.dragon_scale": "龙鳞",
+	"item.cagedmobs.sponge_fragment": "海绵碎片",
+	"item.cagedmobs.nether_star_fragment": "下界之星碎片",
+	"item.cagedmobs.honey_drop": "蜂蜜滴",
+	"item.cagedmobs.star_infused_netherite_ingot": "注星的下界合金锭",
+	"item.cagedmobs.star_infused_netherite_nugget": "注星的下界合金粒",
+	
+	"block.cagedmobs.mobcage": "生物笼",
+	"block.cagedmobs.hoppingmobcage": "漏式生物笼",
+	"block.cagedmobs.star_infused_netherite_block": "注星的下界合金块",
```

```suggestion
	"item.cagedmobs.star_infused_netherite_ingot": "下界之星·下界合金锭",
	"item.cagedmobs.star_infused_netherite_nugget": "下界之星·下界合金粒",
	
	"block.cagedmobs.mobcage": "生物笼",
	"block.cagedmobs.hoppingmobcage": "漏式生物笼",
	"block.cagedmobs.star_infused_netherite_block": "下界之星·下界合金块",
```
先更正为【注星的×××】了，然后如果改成这样呢？会比【灌注下界之星的下界合金×××】更简明而又不像【注星的×××】一样不明所以吗

---

原文没有nether呀，虽然都懂是下界之星（）
个人还是感觉`注星下界合金`就OK的
拉个其他审核来看（）

---

赞同`注星下界合金`

---

> 原文没有nether呀，虽然都懂是下界之星（） 个人还是感觉`注星下界合金`就OK的 拉个其他审核来看（）

那么就 `星·下界合金×××` VS `注星下界合金×××` 吧（话说我的提交好像多了`的`字

---

> CFPA译法`注星下界合金`贴合原文，虽相比其他来源的翻译有所简化，但不至于造成歧义。

行

---

```suggestion
	"item.cagedmobs.star_infused_netherite_ingot": "注星下界合金锭",
	"item.cagedmobs.star_infused_netherite_nugget": "注星下界合金粒",
	
	"block.cagedmobs.mobcage": "生物笼",
	"block.cagedmobs.hoppingmobcage": "漏式生物笼",
	"block.cagedmobs.star_infused_netherite_block": "注星下界合金块",
```